portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,13 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#header{
+    display: flex;
```

Comment:
nit: I think the indentation is a bit off. (2 vs 4 space)

Reply:
Done